event-loop.c

```diff
@@ -10,6 +10,10 @@
 
 #include "kanshi.h"
 
+#ifdef KANSHI_HAS_VARLINK
```

Comment:
All of these need to be `#if` instead of `#ifdef`, because `-DKANSHI_HAS_VARLINK=0` passes the `#ifdef` check.

Would be nice to add a new CI step with `--auto-features=disabled` passed to Meson, so that we can make sure the varlink-less build works fine.